proposal/oscar97123/proposal.rst

```diff
@@ -88,21 +99,27 @@ both.
 Engineering infrastructure
 ==========================
 
-<TO-DO>
+There will be 2 branches in the development. The first branch(master) holds the stable version of the software(with unit test assurance).
+The second branch(development) will be a workstation for developing features, moreover, the unit test will perform in this branch either. 
+Only when the unit test passes and no crashes occurred, then the development branch merge into the master branch as a stable build. 
 
-Describe how you plan to put together the build system, testing framework, and
-documentation.  Show how you will do version control.
+Here's the image showcasing the proposed Version Control.
 
-You may use continuous integration, but it is not required.  If you use it,
-describe how it works in your code development.
+.. image:: https://i.imgur.com/DlnArn8.png
+                :alt: DlnArn8
+
+The project will split into functions for maintainability and the function name will be meaningful to make sure the readability is on point. 
+Besides, there will be comments on complex code.
 
 Schedule
 ========
 
-<TO-DO>
-
-Itemize the work to do and list the work items in a timeline.  Estimate the
-efforts of each item.
+4/11 - 4/18(1 week)  - Research
```

Comment:
You may want to make it a rst list.  For now it is not rendered correctly.  In plaintext mode it's fine.